snoop/management/commands/export-pgp.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+from pathlib import Path
+import base64
+from ... import models
+from ... import emails
+from ... import pgp
+from ...content_types import guess_content_type
+
+class Command(BaseCommand):
+    help = "Export digested pgp .eml files to a zip archive"
+
+    def add_arguments(self, parser):
+        parser.add_argument('where',
+                            help='SQL `where` clause to be ran ' +
+                                 'on the snoop_document table')
```

Comment:
Can we add an example that selects all PGP emails? That's what people would normally run.